### DISCUSSION TITLE
How to automate numbering tables generated by gtsummary and included in a book in quarto? How to make cross-reference to these tables?

### DISCUSSION THREAD

**Discussion by zawkzaw** [quarto-dev/quarto-cli#3042] [Q&A]
Hello!
In a book project by quarto, I produce some tables in chapter 5 using gtsummary package and render the book into docx format But I couldn't make automatic numbering tables like Table 5.1 , Table 5.2 , Table 5.3 , ..., above the tables in docx file.
I tried #| tbl-cap: xxxxxxxxxxxxxx in code chunk and automatic numbering of tables occurred in pdf file but not in docx file.

I couldn't make cross-reference to these tables in Quarto book, either.
 
Could you please advise on these?

Best wishes

Ko Ko Zaw

**Comment by mcanouil** [comment]
For any cross-reference to work you need to define a label `#| label: tbl-*****` or an HTML id `#tbl-*****`.
See <https://quarto.org/docs/authoring/cross-references.html#tables>.
